### ISSUE TITLE
False positive: remark-lint:awesome-git-repo-age

### ISSUE THREAD

**Issue by szarnyasg** [sindresorhus/awesome-lint#190]
I receive a false positive `awesome-git-repo-age` error when running the linter on the [`awesome-duckdb` repository](https://github.com/davidgasquez/awesome-duckdb). The repository is more than a year old.

```bash
npx awesome-lint https://github.com/davidgasquez/awesome-duckdb/
```
```text
✖ Linting

  README.md:195:201
  ✖    1:1    Git repository must be at least 30 days old                             remark-lint:awesome-git-repo-age
...
```

Possible duplicate of #105

**Comment by hstsethi** [comment]
Same issue on my repository. [awesome-j2me](https://github.com/hstsethi/awesome-j2me/)

I recently changed my repository name so maybe that could be a factor but I don't think so if that is.
